core/pom.xml

```diff
@@ -370,5 +370,28 @@
             </activation>
             <!-- not including license-maven-plugin is sufficent to expose default license -->
         </profile>
+        <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>attach-javadocs</id>
+                                <goals>
+                                    <goal>jar</goal>
+                                </goals>
+                                <configuration>
+                                    <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
I would agree but there are many linter errors in the current sources which would prevent build the artifact (at least on Java 8).

Would it be okay to just disable the Javadoc doclint on Java 8, e. g. as described in http://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html?